docs/designers-developers/developers/filters/i18n-filters.md

```diff
@@ -0,0 +1,109 @@
+# i18n Filters
+
+The i18n functions (`__()`, `_x()`, `_n()` and `_nx()`) provide translations of strings for use in your code. As with their server-side PHP equivalents, the values returned by these functions are filterable if you need to override them, using the following filters:
```

Comment:
I think we can get rid of the "As with their server-side PHP equivalents", unless we provide the links to their documentation https://developer.wordpress.org/themes/functionality/internationalization/ and https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/.

Reply:
Agreed. There isn't a "nice" place to link to that covers all of the PHP gettext filters (that I can see), so I've removed that phrase. Thanks.